config/rbac/addresspool_editor_role.yaml

```diff
@@ -0,0 +1,24 @@
+# permissions for end users to edit addresspools.
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: addresspool-editor-role
```

Comment:
What are this and addresspool-viewer-role needed for?

Reply:
this auto generated file when u create an API